### PR TITLE
Development

### PR DESCRIPTION
**Pull request by itsmecv**
@ckylek PR for you to code review. I've refactored logic out of collision.cs and into stancelogic.cs. You'll notice a new empty object in the scene called "stance logic" wherein the script is a component. The DetermineWinners(); function lives there now. It is directly referenced.

### CODE REVIEW

**Comment by itsmecv** [comment]
@ckylek added another commit with charge attack move.

**Comment by itsmecv** [comment]
[Create hold space to charge (at least two seconds), release to attack move](https://trello.com/c/NRkIOUGV/7-create-hold-space-to-charge-at-least-two-seconds-release-to-attack-move)

File: Assets/Scripts/StanceLogic.cs [hunk 1]
```diff
@@ -2,8 +2,74 @@
...
+                else
+                {
+                    Debug.Log(playerStance + " collided with " + enemyStance);
+                    Debug.Log("You Lose!!");
+                }
+
+            }
+    }
     // Start is called before the first frame update
     void Start()
```
Comment (ckylek) [inline Assets/Scripts/StanceLogic.cs:74]: Delete?

File: Assets/Scripts/StanceLogic.cs [hunk 2]
```diff
@@ -2,8 +2,74 @@
...
+                }
+                else
+                {
+                    Debug.Log(playerStance + " collided with " + enemyStance);
+                    Debug.Log("You Lose!!");
+                }
+
+            }
+    }
     // Start is called before the first frame update
```
Comment (kyleknick) [inline Assets/Scripts/StanceLogic.cs:73]: Delete Start() for clean up

**Review by kyleknick — APPROVED** [review]
I'd say it's a work of art but that's overselling art. APPROVED

**Pull request merged**